app/views/settings/external_cert/_form.html.slim

```diff
@@ -0,0 +1,48 @@
+  .panel
+    .panel-default
```

Comment:
You forgot to revert it to `.panel.panel-default`.

Reply:
You are right.  Done now.